spec/features/region_admin/donors_spec.rb

```diff
@@ -0,0 +1,79 @@
+require 'rails_helper'
+
+RSpec.describe "Region Admin Donors" do
+
+  feature "Viewing the list of donors" do
+    let(:boulder) { create(:region) }
+    let(:denver)  { create(:region) }
+
+    let!(:boulder_donor) { create(:donor, region: boulder, name: "Boulder donor site") }
+    let!(:denver_donor)  { create(:donor, region: denver,  name: "Denver donor site") }
+
+    context "as a region admin" do
+      let(:volunteer) { create(:volunteer, regions: [], assigned: true) }
+
+      before do
+        Assignment.add_volunteer_to_region(volunteer, boulder)
+        volunteer.assignments.first.update_attributes(admin: true)
+
+        Assignment.add_volunteer_to_region(volunteer, denver)
```

Comment:
I think I'd have a slight preference for using the `Assignment` factory instead. Just to limit the application code used in the tests.

```ruby
create(:assignment, volunteer: volunteer, region: boulder)
```

What do you think?